www/js/app.js

```diff
@@ -1635,27 +1629,71 @@ document.getElementById('archiveFilesLbl').addEventListener('keydown', function
     }
 });
 
+/** Drag and Drop handling for ZIM files */
+
+// keep track of entrance event so we only fire the correct leave event 
```

Comment:
Very minor, but our house style tries to start comments with a capital letter, so it looks more like a well written sentence (however, only use full stop at end if you need more than one sentence).

Reply:
Gotcha I'll fix with other comments